server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowInstanceService.java

```diff
@@ -471,7 +471,6 @@ public Page<FlowInstanceEntity> listAll(@NotNull Pageable pageable, @NotNull Que
                     TaskType.SHADOWTABLE_SYNC,
                     TaskType.PARTITION_PLAN,
                     TaskType.ONLINE_SCHEMA_CHANGE,
-                    TaskType.ALTER_SCHEDULE,
```

Comment:
@guowl3 may check this

Reply:
how can we process alter schedule ticket if you filter out them?

Reply:
Users need to go to the schedule module to view the tickets.